src/components/GradientButton.tsx

```diff
@@ -10,7 +10,6 @@ import classnames, {
 } from 'classnames/tailwind'
 
 const parentButtonWrapper = classnames(
-  width('w-fit'),
```

Comment:
This style makes it fit the button on small screens. May you test that everything is fine, where that component is used?

Reply:
sure, I tested it. The only two places where we're using it is CTA Buttons and "Submit emal" button, I tested them both and it's works fine on any resolution up to 280px width